recorder/src/main/java/com/focusit/jsflight/recorder/internalevent/httprequest/HttpRecorderFilterBase.java

```diff
@@ -14,12 +14,17 @@
 {
     public static final String ALREADY_FILTERED_SUFFIX = ".FILTERED";
     private static final AtomicBoolean ENABLED = new AtomicBoolean(false);
+    private static final AtomicBoolean NEED_OPEN_OUTPUT = new AtomicBoolean(false);
```

Comment:
is a situation possible when someone has enabled = true and need_open_output = false?
As for me, there should be one variable to determine if output file should be opened or not.